msyncd/AccountsHelper.cpp

```diff
@@ -100,7 +100,6 @@ void AccountsHelper::slotAccountCreated(Accounts::AccountId id)
                 delete syncProfile;
             }
         }
-        delete newAccount;
     }
```

Comment:
Right, that was part of the behaviour change in libaccounts-qt5 version 1.7: manager->account(id); the account returned is owned by the manager (actually, it always was, this isn't the change) but it's unsafe to delete the account since the manager shares instances internally.

So, yeah, the manager has ownership of those instances.
But yes I will change the code to use the other form, to avoid the memory overhead.